src/Exception/FormNotValidException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+use LogicException;
+use Symfony\Component\Form\Form;
+use Throwable;
+
+/**
+ * Class FormNotValidException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle\Exception
+ */
+class FormNotValidException extends LogicException implements Exception
+{
+    /**
+     * @var Form
+     */
+    protected $form;
+
+    /**
+     * ActionUnavailableException constructor.
+     * @param string $message
+     * @param int $code
+     * @param Throwable|null $previous
+     */
+    public function __construct(Form $form, $message = "Form Invalid", $code = 0, Throwable $previous = null)
```

Comment:
long line imho

Reply:
105 chars its long but less than 120 and function have 4 arguments.